seed/main/nv.xml

```diff
@@ -17,8 +17,8 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		</orientation>
 	</layout>
 	<characters>
-		<exemplarCharacters references="R1">[a á ą {ą\u0301} b {ch} {ch’} d {dl} {dz} e é ę {ę\u0301} g {gh} h {hw} i í į {į\u0301} j k {kw} {kʼ} l ł m n o ó ǫ {ǫ\u0301} s {sh} t {tł} {tłʼ} {ts} {tsʼ} {tʼ} w x y z {zh} ʼ]</exemplarCharacters>
+		<exemplarCharacters references="R1">[a á ą {ą\u0301} b {ch} {ch‘} d {dl} {dz} e é ę {ę\u0301} g {gh} h {hw} i í į {į\u0301} j k {kw} {kʼ} l ł m n o ó ǫ {ǫ\u0301} s {sh} t {tł} {tłʼ} {ts} {tsʼ} {tʼ} w x y z {zh} ʼ]</exemplarCharacters>
 		<exemplarCharacters type="numbers">[\- , . % ‰ + 0 1 2 3 4 5 6 7 8 9]</exemplarCharacters>
-		<exemplarCharacters type="punctuation">[\- ‐ – — , ; \: ! ? . … ' ‘ ’ &quot; “ ” ( ) \[ \] § @ * / \&amp; # † ‡ ′ ″]</exemplarCharacters>
+		<exemplarCharacters type="punctuation">[\- ‐ – — , ; \: ! ? . … ' ‘ ‘ &quot; “ ” ( ) \[ \] § @ * / \&amp; # † ‡ ′ ″]</exemplarCharacters>
```

Comment:
Thius is definitenyl wrong, you have changed the paired single quotes U+2018 U+2019 to be U+2018 U+2018

Reply:
Thanks. The test caught this also; my search and replace was getting the characters confused. Please what it looks like now.